src/wp-includes/class-wp-query.php

```diff
@@ -754,6 +754,7 @@ public function fill_query_vars( $query_vars ) {
 	 *     @type array           $tax_query               An associative array of WP_Tax_Query arguments.
 	 *                                                    See WP_Tax_Query->__construct().
 	 *     @type string          $title                   Post title.
+	 *     @type bool            $post_query_cache        Whether cache query. Default false.
```

Comment:
I reckon it's safe to default to `true` with a dev note and getting it in earlyish.

Reply:
Can you please add a new `$post_query_cache` parameter in the query doc block?

Reply:
> I reckon it's safe to default to true with a dev note and getting it in earlyish.

Strongly disagree here. I think this has be a opt in feature, as custom migration scripts will except these queries to be uncached.

Reply:
Done